src/CSConsole/ScriptInteraction.cs

```diff
@@ -31,9 +31,18 @@ public static void Stop(Coroutine coro)
 
         public static void Copy(object obj)
             => ClipboardPanel.Copy(obj);
+        
+        public static void Copy(object obj, int index)
+            => ClipboardPanel.Copy(obj,index,false);
+        
+        public static void CopyToEnd(object obj)
```

Comment:
I think this function is not used and should be removed.

Reply:
I made the CopyToEnd() method for when using the console, just because it makes it easier to add a new item and fill it at the end of the clipboard list. It could probably be removed, I just think it's useful for quick usage of the console.